### DISCUSSION TITLE
Important update

### DISCUSSION THREAD

**Discussion by raosush** [WebClub-NITK/Hacktoberfest-2k21#19] [Announcements]
## Notice
We have restricted access to issues to only collaborators in the repository. In case you wish to discuss anything, feel free to use the **Discussions** option in the repository. Also, if you wish to create an issue, kindly contact the maintainers of the repository @raosush  @muditsinghal @ikjot-2605 @tharun634 
Thank you!

**Comment by khannakshat7** [comment]
@raosush Can you add `Hacktoberfest` topic in the repo so that it can participate in Hacktoberfesst

**Reply by raosush** [reply]
@khannakshat7 We will not be adding the topic to our repository. Your PR submissions will count towards the required number once the maintainers add **hacktoberfest-accepted** label to your PR. Kindly wait for the maintainers to start approving and merging your PRs

**Reply by khannakshat7** [reply]
@raosush The Maintainers are not responding on the PRs and the issues. I have put PR and asked questions a few days back but none of them is responding in the conversation. They don't know the rules to put labels on PR. Please look into it.

**Reply by raosush** [reply]
@khannakshat7 Don't worry, the maintainers will start approving and merging the pull requests. We have certain new guidelines which will be announced soon, which will also clear all the confusion around this. Till then keep hacking, cheers! 🙂

**Reply by khannakshat7** [reply]
@raosush I think your guidelines would take long time to come and Hacktoberfest would end. So please add label to my PR so that it gets counted for Hacktoberfest. It takes 14 days to mature the PR to get counted for Hacktoberfest. Please value my contributions and put the label on PR.

**Reply by khannakshat7** [reply]
![image](https://user-images.githubusercontent.com/32996284/136686557-828b1ba8-6fff-4fbf-8b33-5e233cc9ad22.png)
